src/main/java/module8/Passport.java

```diff
@@ -0,0 +1,40 @@
+package module8;
+
+public class Passport {
+    private String series;
+    private Integer number;
+
+    public Passport(String series, Integer number) {
+        this.series = series;
+        this.number = number;
+    }
+
+    public String getSeries() {
+        return series;
+    }
+
+    public void setSeries(String series) {
+        this.series = series;
+    }
+
+    public int getNumber() {
+        return number;
+    }
+
+    public void setNumber(int number) {
+        this.number = number;
+    }
+
+    @Override
+    public boolean equals(Object o) {
```

Comment:
Incorrect implementation for `equals`. You should also check for `null` and self reference.
I recommend to use IDEA equals and hashcode generator. Try combination `alt + insert`, displayed context menu should contain equals and hashcode generator.

Reply:
Done. Hashcode and equals generator was used